.readthedocs.yaml

```
@@ -0,0 +1,23 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
```

Want to go to 22.04? We just fixed some build issues on 20.04 the other week for all of RTD/Sphinx:
https://github.com/ECP-WarpX/WarpX/blob/23.06/.readthedocs.yml
```suggestion
  os: ubuntu-22.04
```